bedrock/base/templates/base-resp.html

```diff
@@ -167,13 +167,36 @@
     </div><!-- close #outer-wrapper -->
 
     <!--[if IE 9]>
-      <script src="{{ static('js/libs/matchMedia.js') }}"></script>
+      {% javascript 'matchmedia' %}
     <![endif]-->
 
+    {#
+      The logic mess below will be simplified when all pages are refactored
+      to behave well in old IE without page-specific JS.
+    #}
+    {% if force_modern_js != True %}
```

Comment:
The only major instance where I see this needed is on the `/contact/` pages, which probably means refactoring those pages to work well without JS (in IE <= 8) should become a required part of this PR.

I'll remove these conditionals and make a note that the `/contact/` pages need some work.